src/Camp.css

```diff
@@ -1,35 +1,171 @@
-.Center {
-  display: block;
-  width: 50%;
-  margin: 70px auto;
+.Wrapper {
```

Comment:
I suggest you use this convention for naming your CSS elements
https://suitcss.github.io